collector/fixtures/sys.ttar

```diff
@@ -1246,7 +1291,7 @@ Mode: 644
 # ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 Path: sys/devices/platform/applesmc.768/fan1_label
 Lines: 1
-Left side  
+Left side
```

Comment:
This seems like an unrelated change to the fixtures.

Reply:
Should I make a different commit for the trailing white space removal, or is this something that isn't worth committing?

Reply:
Ok. Any further steps on my end?

Reply:
Please revert this part of the change.

Reply:
Done.